ledger/benches/transaction.rs

```diff
@@ -122,15 +142,20 @@ fn execute(c: &mut Criterion) {
                 vm.execute_authorization(
                     execute_authorization.replicate(),
                     Some(fee_authorization.replicate()),
-                    None,
+                    None::<Query<MainnetV0, <LedgerType<_> as ConsensusStorage<_>>::BlockStorage>>,
```

Comment:
Should we add a helper function that returns `None` or the empty query from below?